File: C#/Remove Duplicates/RemoveDuplicates.cs
public class Solution {
    public int RemoveDuplicates (int[] nums) {
        int ptr = 0;
        for (int j = 1; j < nums.Length; j++) {
            if (nums[ptr] != nums[j]) {
                ptr += 1;
                nums[ptr] = nums[j];
            }
        }
        return ptr + 1;
    }
}

File: Python/Remove Duplicates/RemoveDuplicates.py
class Solution:
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if(len(nums) <= 1):
            return len(nums)
        ptr = 0
        for i in range(1, len(nums)):
            if(nums[ptr] != nums[i]):
                ptr += 1
                nums[ptr] = nums[i]
        return ptr+1
